e2e/testsuite/diagnostics/diagnostics.go

```diff
@@ -25,11 +23,10 @@ const (
 
 // Collect can be used in `t.Cleanup` and will copy all the of the container logs and relevant files
 // into e2e/<test-suite>/<test-name>.log. These log files will be uploaded to GH upon test failure.
-func Collect(t *testing.T, dc *dockerclient.Client, cfg testconfig.ChainOptions) {
+func Collect(t *testing.T, dc *dockerclient.Client, chainNameA, chainNameB string, debugModeEnabled bool) {
```

Comment:
What's the reasoning for the API change? Seems like it might be useful for `Collect` to be able to reference the `DebugConfig`

Reply:
we had to change this also due to circular import issues: otherwise the `diagnostics` package would have to import this `testsuite` package (previously `testconfig` package`), which would then lead to circular import when `testsuite` imports this function

Reply:
I believe the reasoning here is also to avoid a circular dependency